testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -51,6 +51,10 @@
     SUPPORTS_RENAME_TABLE,
     SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS(SUPPORTS_RENAME_TABLE),
 
+    SUPPORTS_ADD_COLUMN,
+    SUPPORTS_DROP_COLUMN,
+    SUPPORTS_RENAME_COLUMN,
```

Comment:
Are there any connectors which support partially? Why is it different from `SUPPORTS_ALTER_TABLE` (or `SUPPORTS_ALTER_COLUMN`).

Reply:
`SUPPORTS_ALTER_TABLE` would be too broad -- there are many things under `ALTER TABLE` umbrella, so partial support should be expected.